scripts/build_pkgdown.R

```diff
@@ -29,7 +33,30 @@ for (pkg in packages) {
       stop(paste("Package directory does not exist:", pkg))
     }
     setwd(pkg) 
-    pkgdown::build_site() 
+    pkgdown::build_site(
+      pkg = ".",
+      override = list(
+        repo = list(
+          url = list(
+            source = paste0("https://github.com/PecanProject/pecan/blob/", 
+                            branch, "/", pkg)
+          )
+        ),
+        template = list(
+          bootstrap = 5,
+          includes = list(
+            before_navbar = paste0(
+              "<style>",
+              ".custom-back-button{position: absolute; top: 10px;left:10px; z-index: 1000; background-color: #f5f5f5; color: #000;border:none }.navbar-brand{ margin-left: 50px }",
+              "</style>\n",
+              "<button onclick='window.location.href=\"../index.html\"' ",
+              "class='custom-back-button btn btn-primary'>",
+              "← All Packages</button>"
+            )
```

Comment:
```suggestion
            before_navbar = "<a href=\"../index.html\" style=\"padding: 0em 1em\">← Up</a>"
```

Reasons for this change:

1. Using an ordinary link (instead of a button that only acts via onclick) allows standard navigation such as cmd-clicking to open in a new tab
2. In narrow windows the absolute positioning leads to the button covering the package name
3. This link is technically an "up one level", which is not always a direct link to the all-packages page. When you're on a function reference page or a package article, one level up is to the readme for that package. I'm fine with that behavior if it's labeled as such.